scripts/explorationregions.gd

```diff
@@ -138,7 +138,7 @@ var database = {
 		code = 'mountaincave',
 		name = 'Mountain Caves',
 		description = "You step onto the damp cave floor. These underground systems serve as home to many various beasts and semisentient creatures.",
-		enemies = [{value = 'oozesgroup', weight = 2}, {value = 'solospider', weight = 2}, {value = 'goblingroup', weight = 4},{value = 'treasurechest', weight = 1}],
+		enemies = [{value = 'oozesgroup', weight = 20},{value = 'goblinattackers1', weight = 8},{value = 'goblinattackers2', weight = 1},{value = 'delffew', weight = 1},{value = 'delfattackers1', weight = 1},{value = 'delfattackers2', weight = 1},{value = 'delfvsgob1', weight = 1},{value = 'solospider', weight = 20},{value = 'goblingroup', weight = 40},{value = 'treasurechest', weight = 10}], #ralphA
```

Comment:
You intend to change the weight of a bunch of these encounters so much?

Reply:
yes, generally I've added rare encounters and rather than put a bunch of decimals for weighting on them, I multiplied all the vanilla encounters by 10 or so.  The weightings are intentional and I think well balanced given they mostly involve rare race capture opportunities.